.github/workflows/wingsdk-mutation.yml

```diff
@@ -20,7 +20,7 @@ env:
   CI: "true"
 
 jobs:
-  build:
+  mutate:
```

Comment:
@MarkMcCulloh - maybe add a comment here in a subsequent PR, just to explain why we called it `mutate` instead of `build` and to prevent someone from refactoring it to `build` again to be consistent with names across our ymls...